.github/CONTRIBUTING.md

```diff
@@ -8,4 +8,15 @@ Welcome to contributing in **Open Source Programs**!
 - When submitting a PR, make your PR as descriptive as possible, follow the template, add screenshots and relevant information.
 - When you are finished with your work, please squash your commits otherwise we will squash them on your PR. This will help maintain a clean commit history.
 - For in-code documentation, follow [this guide](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings).
-- The full guidelines for python can be found [here](https://github.com/google/styleguide/blob/gh-pages/pyguide.md).
\ No newline at end of file
+- The full guidelines for python can be found [here](https://github.com/google/styleguide/blob/gh-pages/pyguide.md).
+
+# General Guidelines
+
+- If you’re just getting started work on an issue labeled “First Timers Only” in any project. You can refer additional resources available on our [website](http://systers.io/).
+- In an active repository (not an archived one), choose an open issue from the issue list, claim it in the comments or ask around in [zulip](https://anitab-org.zulipchat.com/), and a maintainer will assign it to you.
+- After approval you must make continuous notes on your progress in the issue while working. Remember to update your progress in comments after every 3 days,otherwise the maintainer can reassign the issue.
```

Comment:
```suggestion
- After approval you must make continuous notes on your progress in the issue while working. Remember to update your progress through comments after every 3 days, otherwise the maintainer can reassign the issue.
```